test/ssl-tests/17-renegotiate.conf.in

```diff
@@ -19,7 +19,8 @@ our @tests = (
     {
         name => "renegotiate-client-no-resume",
         server => {
-            "Options" => "NoResumptionOnRenegotiation"
+            "Options" => "NoResumptionOnRenegotiation",
+            "MaxProtocol" => "TLSv1.2"
```

Comment:
Not having it causes the tests to fail. The reason is this check in `ssl_choose_server_version()`

```
    switch (server_version) {
    default:
        if (version_cmp(s, client_version, s->version) < 0)
            return SSL_R_WRONG_SSL_VERSION;
```
When renegotiating, `s->version` is set to TLS1.3, but we always set the legacy version to TLS1.2, so this falls over. However, renegotiation is not allowed in TLS1.3. I've just not got as far as actually disabling it yet. So the reneg tests are restricted to only go up to TLS1.2. I've also just added a TODO to remind us to go back and disable TLS1.3 reneg.